stable/admin/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,6 @@
+{{- if eq (include "defaulttrue" .Values.nuodb.addServiceAccount) "true" }}
```

Comment:
Should this be conditionalized on `nuodb.serviceAccount` being defined, that way we do not create a service account that does not actually get used?

Reply:
This makes sense. I will fix it.